test/mixtures.jl

```diff
@@ -133,4 +133,10 @@ using Base.Test
     @test xx*δx + Taylor1(typeof(δx),5) == δx + δx^2 + Taylor1(typeof(δx),5)
     @test xx/(1+δx) == one(xx)
     @test typeof(xx+δx) == Taylor1{TaylorN{Float64}}
+
```

Comment:
Maybe you should convert `X` and/or `Y` to `Taylor1{TaylorN{Float64}}`s and include another test for that.

Reply:
Great, I'll do this in a moment!